CoreScriptsRoot/CoreScripts/DeveloperConsole.lua

```diff
@@ -1,1434 +1,35 @@
-
-local useNewConsole = false
-pcall(function()
-	useNewConsole = settings():GetFFlag("NewInGameDevConsole")
-end)
-
-if useNewConsole then
-	local DeveloperConsoleModule;
-	local function RequireDeveloperConsoleModule()
-		if not DeveloperConsoleModule then
-			DeveloperConsoleModule = require(game:GetService("CoreGui"):WaitForChild('RobloxGui').Modules.DeveloperConsoleModule)
-		end
-	end
-
-	local screenGui = script.Parent:FindFirstChild("ControlFrame") or script.Parent
```

Comment:
Are lines 15-39 not necessary with the flag turned off? It looks like they are within the if-block.